starting-oss-projects.md

```diff
@@ -1,34 +1,155 @@
-# Tools
-## Credential scanner
-## Reuse tool
-## CLA Assistant
-# Company-based vs. foundation-driven
-## When to select which approach
-## Leveraging a foundation (such as Eclipse)
-# Lifecycle mgmt
-## Different project levels (incubator etc.)
-## Quality criteria (CII Badge …)
-## Day-2-ops (CI/CD, security, license scanning, …)
-## Community mgmt (-> TODO group paper)
-# Analytics
+
+### Code of conduct
+
+### Templates (issues, PR’s)
+
+### “Health files” (-> GitHub health files)
+
 ## Tools
-# Sunsetting a project (-> TODO group paper)
+
+### Credential scanner
+
+### Reuse tool
+
+### CLA Assistant
+
+## Company-based vs. foundation-driven
+
+### When to select which approach
+
+### Leveraging a foundation (such as Eclipse)
+
+## Lifecycle mgmt
+
+The life cycle of an open source project describes the stages in which the project evolves, from its conception to its retirement or end of life stage. Typically, a project originates to solve a specific problem. It may become obsolete either because the problem does not exist anymore or because other projects are better suited to solve the problem. The figure below shows the different stages an open source project may undergo.
+
+![Typical lifecycle of an open source project](./img/OSS-lifecycle.png)
+
+### Planning or Concept Phase
+
+This is the starting point of every open source project. It can also be referred to as the “initiation phase”. Normally, at this stage, only an idea exists or a specific problem has been identified which requires solution. In this phase, the open source project typically has the following characteristics:
+
+* The problem that the project intends to solve has been clearly defined
+* There is either no source code available yet or the source code is only internally available. In the first case, the project only exists as idea; in the second case, the project may have been started as an company-internal project and has not been published yet
+* Possibly, the idea has been already shared with the community to get feedback. However, note that this sharing such ideas that so far have only been discussed company-internally requires approval in advance.
+
+Before starting a project, it is reasonable to get answers to the key questions:
+
+* Is it possible to join efforts with an existing open source project?
+* Can we launch and maintain the project using the OSS model?
+* What constitutes success? How do we measure it?
+* Can we financially sponsor the project? Do we have an internal executive champion?
+* Will the project be able to attract outside enterprise participation (from the start)?
+* Is there enough external interest to form and grow a developer community?
+
+(Source: [Linux Foundation](https://www.linuxfoundation.org/en/resources/open-source-guides/starting-an-open-source-project/))
+
+In addition, the following aspects should be considered in the planning phase:
+
+* What is the goal of the project and will it solve the problem?
```

Comment:
here also choosing an appropriate license can be added, because the license depends on the goal which shall be achieved

Reply:
will change that